### PR TITLE
Claudia

### PR DESCRIPTION
**Pull request by claudiagreen**

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -1,11 +1,30 @@
...
+
   </head>
   <body>
+    <div id="github-search" ng-controller="gitHubUserController as controller">
+
+      <input type="text" name="name-search" value="" ng-model="searchTerm">
+      <button ng-click="controller.searchUsers(searchTerm)">Search users</button>
+      <ul id="user-list">
+        <li ng-repeat="users in controller.users | filter:searchTerm">
+        <!-- <li ng-repeat="users in controller.users"> -->
```
Comment (AnnemarieKohler) [inline app/index.html]: remove the comments

File: app/js/controllers/userListController.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+gitHubUserList.controller('gitHubUserController', ['UserFactory', function(UserFactory) {
+  var self = this;
+  this.users=[{username: "claudia", avatar: "testurl", followers: "1000", repos: "1"},
+              {username: "andrew", avatar: "testurl", followers: "1", repos: "5"},
+              {username: "lorenzo", avatar: "testurl", followers: "100", repos: "512"},
+              {username: "poo", avatar: "testurl", followers: "121", repos: "95"}]
```
Comment (AnnemarieKohler) [inline app/js/controllers/userListController.js]: please don't use words like poo

File: test/e2e/userListFeature.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  //   browser.get('/');
+  //   var guy = $$('#user-list img').first();
+  //   expect(guy).toMatch(testurl)
+  // })
+
+});
+
+
+
+write the test
```
Comment (anne-lydon) [inline test/e2e/userListFeature.js:16]: should this be commented out?

**Comment by anne-lydon** [comment]
:shipit:

**Comment by robota-x** [comment]
:shipit:

**Pull request merged**